src/Node.hs

```diff
@@ -77,14 +77,16 @@ data ListenerAction packing m where
   -- | A listener that handles an incoming bi-directional conversation.
   ListenerActionConversation
     :: ( Packable packing snd, Unpackable packing rcv, Message rcv )
-    => (LL.NodeId -> ConversationActions snd rcv m -> m ())
```

Comment:
Yeah, and then we had no time to rollback till now. @georgeee do you think some use case may appear again, or we'd better rollback?